ecl/resource_ecl_network_load_balancer_v2.go

```diff
@@ -0,0 +1,1384 @@
+						},
+
+						"log_facility": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"LOCAL0", "LOCAL1", "LOCAL2", "LOCAL3", "LOCAL4", "LOCAL5", "LOCAL6", "LOCAL7",
+							}, false),
+						},
+
+						"log_level": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+
+						// Call Syslog's Delete/Create API when updated.
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						// Call Syslog's Delete/Create API when updated.
+						"port_number": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							Computed:     true,
+							ValidateFunc: validation.IntBetween(1, 65535),
+						},
+
+						"priority": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							ValidateFunc: validation.IntBetween(0, 255),
+						},
+
+						"status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"tcp_logging": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"NONE", "ALL",
+							}, false),
+						},
+
+						// Call Syslog's Delete/Create API when updated.
+						"tenant_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+
+						"time_zone": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"GMT_TIME", "LOCAL_TIME",
+							}, false),
+						},
+
+						// Call Syslog's Delete/Create API when updated.
+						"transport_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"UDP",
+							}, false),
+						},
+
+						"user_configurable_log_messages": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"YES", "NO",
+							}, false),
+						},
+					},
+				},
+			},
+
+			"tenant_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+
+			"user_password": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+
+			"user_username": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+
+	return result
+}
+
+func resourceNetworkLoadBalancerV2Create(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	networkClient, err := config.networkV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("error creating ECL network client: %s", err)
+	}
+
+	// Get plan and check response whether the plan is enabled or not.
+	var i interface{}
+	var ok bool
+	if i, ok = d.GetOk("load_balancer_plan_id"); !ok {
```

Comment:
requiredなのでGetOKを使用する必要はありません。
```suggestion
	planID := d.Get("load_balancer_plan_id").(string)
```

Reply:
Getを使用するようにし、更に使用箇所は1箇所だけだったため事前の宣言を無くしました。